File: app/src/main/java/com/lemonade/dima/database/MessageDao.java
package com.lemonade.dima.database;

import android.arch.persistence.room.Dao;
import android.arch.persistence.room.Insert;
import android.arch.persistence.room.Query;

import com.lemonade.dima.model.Message;

import java.util.List;

import io.reactivex.Single;

@Dao
public interface MessageDao {
    @Query("SELECT * FROM message")
    Single<List<Message>> getAll();

    @Insert
    void insert(Message employee);
}

File: app/src/main/java/com/k4dima/chat/app/db/MessageDao.kt
package com.k4dima.chat.app.db

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query

import com.k4dima.chat.app.model.Message

import io.reactivex.Single

@Dao
interface MessageDao {
    @Query("SELECT * FROM message")
    fun all(): Single<MutableList<Message>>

    @Insert
    fun insert(employee: Message)
}